Input leap cannot be built on macports

### What happened?

Running `./clean_build.sh` results in the following output:

```
/opt/local/bin/cmake
Modifying environment for Barrier build...
/opt/local/bin/port
Detected Macports
done
/opt/local/bin/ninja
Starting Input Leap Debug build...
CMake Error: Could not create named generator Ninja -DCMAKE_BUILD_TYPE=Debug  -DCMAKE_OSX_SYSROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -DCMAKE_OSX_DEPLOYMENT_TARGET=10.9
```


### Version

From Git HEAD or commit (specify below)

### Git commit hash (if applicable)

_No response_

### If applicable, where did you install Barrier from?

Git

### What OSes are you seeing the problem on? (Check all that apply)

macOS

### What OS versions are you using?

macOS 10.14.4.

### Relevant log output

_No response_

### Any other information

_No response_

--- a/clean_build.sh
+++ b/clean_build.sh
@@ -42,5 +42,7 @@ mkdir build || exit 1
 cd build || exit 1
 echo "Starting Input Leap $B_BUILD_TYPE build..."
 "$B_CMAKE" "$B_CMAKE_FLAGS" .. || exit 1
+echo "Starting Barrier $B_BUILD_TYPE build..."
+"$B_CMAKE" $B_CMAKE_FLAGS .. || exit 1
 "$B_CMAKE" --build . --parallel || exit 1
 echo "Build completed successfully"

